helpdesk_mgmt_project/tests/test_helpdesk_ticket.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.helpdesk_mgmt.tests import test_helpdesk_ticket
+
+
+class TestHelpdeskTicketProject(test_helpdesk_ticket.TestHelpdeskTicket):
```

Comment:
@pedrobaeza , do the tests have to build the scenario independently ?, or can I use this type of license to avoid having to create data?

Reply:
You can do that, but maybe a refactoring is needed in `helpdesk_mgmt` module  for not repeating tests. See this example:

- Base class with only setup:

https://github.com/OCA/l10n-spain/blob/b7ce5aedbdefbe4f4cf3d5ad975bec153269f7ab/l10n_es_aeat_mod303/tests/test_l10n_es_aeat_mod303.py#L12

- Real test class in the same module that performs the test of it:

https://github.com/OCA/l10n-spain/blob/b7ce5aedbdefbe4f4cf3d5ad975bec153269f7ab/l10n_es_aeat_mod303/tests/test_l10n_es_aeat_mod303.py#L290

- Extension module that uses the same test class base for using the data in setup:

https://github.com/OCA/l10n-spain/blob/b7ce5aedbdefbe4f4cf3d5ad975bec153269f7ab/l10n_es_aeat_vat_prorrate/tests/test_l10n_es_vat_prorrate.py#L8

Reply:
fantastic! Thanks!